src/docs/WantToHelp/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Want to Help ?'
+order: 8
+description: 'Discover how you could contribute to Mozaic'
+---
+
+## Want to help ? Here are things that you can work on with us
+
+### Help us build/migrate components
+
+- Dialog component
+- Tabs component
+- Tooltips component
+- Breadcrumb component
+- Spinner / loader component
+- Pagination component
+- Carousel component
+- Card component
+- Product Tile Component
+- Overlay Layout
+- Toaster layout
+
```

Comment:
pattern, not component

Reply:
pas dans ce cas, nous avons bien des composants et des layouts.